### DISCUSSION TITLE
Pausing roll mode

### DISCUSSION THREAD

**Discussion by dzid26** [OpenHantek/OpenHantek6022#374] [Q&A] [answered]
I have trouble pausing in roll mode consistently. Sometimes it pauses quite quickly, sometimes almost the whole frame is scrolled before it pauses. 

So I am forced to predict to pause ahead of time, but it is not always the same delay.

Ideally, I would like to pause immediately.

**Comment by Ho-Ro** [comment, marked as answer]
Did you check #375 and latest [3.3.3-rc1](https://github.com/OpenHantek/OpenHantek6022/releases/tag/3.3.3-rc1)?

**Comment by dzid26** [comment]
I see.
It works well now!

Thank you
